### ISSUE TITLE
文藝春秋のドメイン配下に mozilla.jp のコピーサイトが設置されています

### ISSUE THREAD

**Issue by Guest126** [mozilla-japan/www.mozilla.jp#9]
DuckDuckGoで「[Mozilla Japan](https://duckduckgo.com/?q=Mozilla+Japan)」を検索すると、一般社団法人時代の Mozilla Japan サイトが検索結果にちらほら表示されます。

- http://number-blog.bunshun.jp/blog/
- http://test.number.bunshun.jp/blog/
- http://test.number.bunshun.jp/about/

ベースドメインの [bunshun.jp](https://bunshun.jp/) は週刊文春などで知られる株式会社文藝春秋の所有するドメイン名です。

> Domain Information: [ドメイン情報]
> [Domain Name]                   BUNSHUN.JP
> 
> [登録者名]                      株式会社文藝春秋
> [Registrant]                    BUNGEISHUNJU Ltd. 

サブドメインに含まれる"number"という文字は文藝春秋の発行するスポーツ総合雑誌「Sports Graphic Number」（通称「Number」）とも関係ありそうな気がします。

Github にアーカイブされている [mozilla-japan/mozilla.jp: Mozilla Japan コミュニティポータルサイトのソース](https://github.com/mozilla-japan/mozilla.jp)をテスト目的で設置したもののように見受けられます。

ログイン機能などはなかったと思うので、悪用の可能性は低いと思いますが、看過することもできないので、ここに報告させていただきます。

**Comment by hmatrjp** [comment]
確かに、サブドメインを1つ除いた https://number.bunshun.jp/ は、雑誌 Number のサイトのようですね。
編集部の誰か、またはサイト構築を受託した誰か、がテスト目的で使用して一般的な検索サイトに出ていることに気づいていない可能性が大なので、Number のサイトの[下の方にある問い合わせリンク](https://number.bunshun.jp/list/contact)
から問い合わせてみました。

> Numbers のドメイン配下のサイトとして、例えば下記が一般検索サイトの結果として表示される（DuckDuckGoから Mozilla Japan をキーワードとして検索した場合）のですが、これは Mozilla のサイトのミラーサイトでしょうか？
>     http://number-blog.bunshun.jp/blog/
> 
> 他にも、number.bunshun.jp のサブドメインで
>     http://test.number.bunshun.jp/blog/
>     http://test.number.bunshun.jp/about/
> が、一般からアクセスできるサイトとして公開されています。

**Comment by Guest126** [comment]
しばらく動きがないので、[Mozilla商標ガイドライン](https://www.mozilla.org/en-US/foundation/trademarks/policy/)違反ということで、[商標侵害報告フォーム](https://www.mozilla.org/en-US/about/legal/defend-mozilla-trademarks/)から上記の内容を英語に翻訳してMozillaに送信しました。

余談ですが、Mozilla Japanの商標等は現在誰が使用許諾を受けているのでしょう？ Mozilla JapanからWebDINO Japanに改組された際にそのまま引き継ぎがされたのか、Mozilla Foundationに巻き取られたのか、コミュニティ側に付与されたのか、よくわかりませんでした。

**Comment by hmatrjp** [comment]
> Mozilla Japanの商標等は現在誰が使用許諾を受けているのでしょう？

Webdino Japan の方に聞いてみるのが一番と思います、　どなたに尋ねるのが良いでしょう？　@dynamis
